some text on Known Good Values from LL (#79)
ietf-rats-wg/architecture: draft-ietf-rats-architecture.md

by applying an Evidence Appraisal Policy to assess the trustworthiness of the Attester,
and generates Attestation Results for use by Relying Parties.  The Evidence Appraisal Policy
might be obtained from an Endorser along with the Endorsements, or might be obtained

> via some other mechanism such as being configured in the Verifier by an

Reviewer: Why the change in this line?

administrator.

> For example, for some claims the Verifier will compare the claims in the
> Evidence to known-good-values or reference values.
> In many cases these are hashes of files, SW or memory regions.
> In other cases they might be expected SW or HW versions.
> In other cases, they may be something else.

Reviewer: ```suggestion
For example, for some claims the Verifier might compare the values of claims in the
Evidence to known-good values or reference values.
```

It's confusing to say that for some (arbitrary) claims, one might compare the value to something unrelated like a software version.

> The actual data format and semantics of a known-good-value are specific to

Reviewer: ```suggestion
The actual data format and semantics of a known-good value are specific to
```

claims and implementations.
There is no general purpose format for them or general means for comparison defined in this architecture document.

> 

Reviewer: ```suggestion
Similarly, for some claims the Verifier might check the values of claims in the Evidence for membership in a set, or against a range of values, or against known-bad values such as an expiration time.
```

> These known-good-values may be conveyed to the Verifier as part of an Endorsement or as part of Appraisal Policy or both as these are the two input paths to the Verifier.

Reviewer: This statement isn't specific to known-good values, it can apply to known bad values, ranges, sets, etc.

> These known-good-values may be conveyed to the Verifier as part of an Endorsement or as part of Appraisal Policy or both as these are the two input paths to the Verifier.

Author: ```suggestion
These reference values may be conveyed to the Verifier as part of an Endorsement or as part of Appraisal Policy or both as these are the two input paths to the Verifier.
```
